recipes/paprika/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "paprika" %}
+{% set version = "0.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/slochower/pAPRika/archive/v{{ version }}.tar.gz
+  sha256: 22a97f6362b6ef9aea9210bd811e2c713dc29158bb61d17af9568a052778cf49
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - setuptools
+    - pymbar
+    - mdtraj
+
+test:
+  requires:
+    - pytest
+  imports:
+    - paprika
+
+about:
+  home: https://github.com/slochower/pAPRika
+  license: BSD-3-Clause 
+  summary: pAPRika is a toolkit for setting up, running, and analyzing free energy molecular dynamics simulations.
```

Comment:
Can you add the `doc_url` and `dev_url` fields.

Reply:
Done

Reply:
Noted. Handling full deployment is a bit beyond what we set out to do with the Cookiecutter, and keeping in full sync with c-f, anaconda, and PyPi would be a daunting task and very much beyond what we aim to do, so we may just update our docs better. But yay for the c-f Linter which take care of such things!

Reply:
Agreed that full compatibility would be daunting and time consuming. Maybe worthwhile to just note check `examples/meta.yaml` for additional requirements or tips, etc.